Final Project/Server/fiimentor/controllers/authController.ts

```diff
@@ -1,29 +1,45 @@
 import HttpStatus from "http-status-codes";
+import {createConnection} from 'typeorm';
+import {getConnection} from "typeorm";
+import {UserRepository} from '../Repositories/UserRepository'
 
 import {createToken} from '../utils';
+import {create} from "domain";
 
 async function login(req: any, res: any){
     try {
-        // Mock user
-        const user = {
-            id: 1,
-            username: 'teodor',
-            email: 'teodor@gmail.com'
-        };
-
         const username = req.body.username;
         const password = req.body.password;
 
-        //serach in the database if user is existent
-
+        createConnection().then(async connection => {
+            const userRepository = new UserRepository(connection);
+            const userByUserName = await userRepository.getByUsername(username);
+            await connection.close();
+            console.log(userByUserName);
+            if (userByUserName[0] == undefined || userByUserName[0] == null){
```

Comment:
schimbi tipul returnant din metoda getByUsername in User[] si aici verifici cu !userByUserName.length